### ISSUE TITLE
OpenSUSE 42.3 duplicate fstab entries

### ISSUE THREAD

**Issue by vholer** [OpenNebula/addon-context-linux#104]
After first boot, duplicate rootfs fstab entry is created:

```
cat /etc/fstab
/dev/vda1  /       ext4    defaults        1 1
/dev/vda1 / ext4 defaults 1 1
```

which leads to failure and few minutes delay on subsequent boots:

![image](https://user-images.githubusercontent.com/414850/36376274-c3d34642-1572-11e8-81d3-f913e1bf64e2.png)

**Comment by vholer** [comment]
Even if fstab is OK, udev settle blocks the boot:
![image](https://user-images.githubusercontent.com/414850/36475275-6b40a978-16fa-11e8-9a63-c317646f04c0.png)

**Comment by vholer** [comment]
* The duplicate fstab entries will be fixed in the Marketplace appliance. The initial fstab will be generated on very first boot, there won't be any in the image.
* The udev settle timeout was fixed in aa3a9f2b4acc21c3e692b4e245ddefdafa7b7fdd by using non-blocking systemctl starts.
